guides/user-guide.md

```diff
@@ -0,0 +1,67 @@
+
+The consultant team keep an eye open for email requests. Normally, you will get an acknowledgement or more detailed reply within two working days.
+
+To help us help you, please answer these questions in your email request, as best you can:
+
+1.  Who is the work for?
+2.  Why is it important (ie business case)?
+3.  What happens if the work is not done (or not done on time)?
+4.  What is the requirement (old-style accreditation on an existing contract, guidance or advice, review of proposed approach,...)?
+5.  What skills or experience do you believe is required to do the work (known skills/experience, or predicted skills/experience)?
+6.  When is the next milestone for the project, where cyber consulting input/involvement is requested?
+
+## How we handle requests for help
+
+Each working day, all new requests are reviewed.
+
+If the request is not appropriate for the team, a reply is sent promptly, explaining why it would be better to talk with a different team. We will usually recommend a more appropriate team in such cases, and provide contact details for them.
+
+If the request is appropriate for the consultants, some initial estimates are immediately carried out, such as:
+
+- Does the request affect many people or services across the MoJ? The wider the range, the greater the impact of the request.
+- Does the request require a small or large number of people, or time, to address? Some requests can be answered promptly by an email. Other requests might require a team of 2 or 3 people to be assigned for a 2 month project.
+- Does the request need certain skills or experience? This helps determine which consultant might be best placed to help.
+
+The answers to these questions help the team to decide whether a consultant can help immediately or not.
+
+If no consultant is available, the request is put on a 'next day refresh', meaning that it is considered again on the next working day. If a request stays in the 'next day refresh' state for too long - typically a week - it is automatically escalated to higher management for a review and decision.
+
+## What happens next
+
+If your request is not appropriate for the consulting team, you'll be told immediately after the initial assessment concludes.
+
+If your request is appropriate for the consulting team, you'll be contacted directly by the assigned consultant, who will engage directly with you to start providing the help you need.
+
+## If things go wrong...
+
+If you disagree with a decision that your request is not appropriate for the consulting team, or there is some other problem, please contact the team again through the [cyberconsultancy@digital.justice.gov.uk](mailto:cyberconsultancy@digital.justice.gov.uk) email address. If you would prefer a different escalation route, please contact <img src="https://s3-eu-west-2.amazonaws.com/intranet-prod-storage-1dvcquh7kophi/uploads/2019/02/7d4181b72d6583785cd81bc9726d50c9.gif" alt="Alternative contact"> directly.
```

Comment:
I feel like this should not be Jon's own email, more of a 'role' email address (aliases to the incumbent) `ciso@digital.justice.gov.uk` etc

Reply:
Agree with Joel!